testsuite/forge.py

```diff
@@ -1665,5 +1643,207 @@ def tail(
     ).unwrap()
 
 
+DEFAULT_CONFIG = "forge-wrapper-config"
+DEFAULT_CONFIG_KEY = "forge-wrapper-config.json"
+
+
+# All changes to this struct must be backwards compatible
+# i.e. its ok to add a new field, but not to remove one
+class ForgeConfigValue(TypedDict):
+    enabled_clusters: List[str]
```

Comment:
nit: can we also specify the region each cluster is in?
Also might be useful to be able to specify which cloud, in case there are different ways to auth against them. Not currently used, and can be done in a separate PR / when needed.